content/api/template-language.apib

```diff
@@ -465,6 +465,18 @@ The template language identifies links by the fact that they start with a protoc
 
 To use links that are entirely stored in variables processed correctly, render them as [dynamic content](#header-dynamic-content).
 
+### Link Extraction
```

Comment:
I think a section title like "Links in Loops and Conditionals" would be more appropriate